### PR TITLE
Fix #4274: When collecting linked aliases use SymbolFlags.Alias to capture local aliases as well

### LINKED ISSUE #4274: unexpected definition file generated with es6 import export expression.
unexpected definition file generated with es6 import export expression.

expected
```
$ tsc --module commonjs --declaration index.ts
$ cat index.d.ts
import { bar } from "./utils";
export { bar };
```

actual
```
$ tsc --module commonjs --declaration index.ts
$ cat index.d.ts
export { bar };
```

index.ts
```
import {foo, bar, Buzz} from "./utils";

foo();
let obj: Buzz;
export {bar};
```

utils.ts
```
export function foo () {}
export function bar () {}
export interface Buzz {}
```

this example is exec in tsc 1.5.3 and master/HEAD.

repro code.
https://gist.github.com/vvakame/92a86decdcccdf8dfbf2

### PR DESCRIPTION
**Pull request by mhegazy**
Fix #4274

### CODE REVIEW

File: src/services/formatting/rules.ts [hunk 1]
```diff
@@ -725,6 +726,7 @@ namespace ts.formatting {
                 case SyntaxKind.ConstructSignature:
                 case SyntaxKind.CallExpression:
                 case SyntaxKind.NewExpression:
+                case SyntaxKind.ExpressionWithTypeArguments:
```
Comment (DanielRosenwasser) [inline src/services/formatting/rules.ts]: Why is this part of this change?

Comment (mhegazy) [inline src/services/formatting/rules.ts]: no idea.

**Comment by DanielRosenwasser** [comment]
What other issue does this

**Comment by mhegazy** [comment]
```ts
import {bar} from "mod";
export = bar;
```

**Comment by sheetalkamat** [comment]
:+1:

**Pull request merged**